ext_mongodb.php

```diff
@@ -746,7 +868,7 @@ public function getCode() : string
 
 	public function getScope() : mixed
 	{
-		if ( $this->scope !== NULL )
+		if ( isset( $this->scope) && $this->scope !== NULL )
```

Comment:
I've changed my code to always include "scope". You will need to do this as well now.